File: app/src/main/java/com/example/myapplication/Task.java
package com.example.myapplication;

public class Task {
    public static void main(String[] args) {
        Person person = null ;
        Person person1 = new Person();
                person1.name = "rajan";

        Person person2 = new Person();
        person1.name = "rajan";
        System.out.printf(String.valueOf(person1 == person2));

        if (person != null) {
            System.out.println(person.name);
        }
    }
}
class Person{
    String name = "rajan";

}

File: libKotlinModule/src/main/java/com/example/libkotlinmodule/Task.kt
package com.example.libkotlinmodule

class Task {
}

fun main() {
    var person: PersonClass? = PersonClass("rajan",22)

    println(person.convertToString())
}
class PersonClass(var name: String,var age: Int){

}



fun PersonClass?.convertToString(): String{
    if(this == null) return "null"
    return this.name
}

class UserClass(var name: String){

}

fun PersonClass?.getUser(): UserClass? {
    if(this == null) return null
    var user = UserClass(this.name);
    return user
}